fabfile.py

```diff
@@ -81,6 +86,10 @@ def update():
     run('systemctl restart uwsgi.service')
     run('systemctl restart nginx')
 
+    # Update secrets
+    # To do
```

Comment:
Please use "TODO" as it it recognized by at least Pycharm IDE.

Reply:
Okay, thanks!